template.tpl

```diff
@@ -1396,6 +1408,36 @@ const generateConfiguration = () => {
   return initOptions;
 };
 
+const getAllUserProps = (data) => {
```

Comment:
Why is the logic for user properties very different compared to event properties?

Reply:
The format for identify is different. 
`
        ['add', 'someUserProp', 1],
        ['add', 'someOtherUserProp', 2],
        ['prepend', 'anotherUserProp', 'someValue']
`

Reply:
ah interesting. Thanks